plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestSnowflakeConnectorTest.java

```diff
@@ -66,10 +66,8 @@ protected SqlExecutor onRemoteDatabase()
     protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
         return switch (connectorBehavior) {
-            case SUPPORTS_AGGREGATION_PUSHDOWN -> true;
```

Comment:
@chenjian2664 I reverted your change as SUPPORTS_AGGREGATION_PUSHDOWN and SUPPORTS_PREDICATE_PUSHDOWN are true by default.